zhazam/Modules/StartGameTimer/StartGameTimerViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  StartGameTimerViewController.swift
+//  zhazam
+//
+//  Created by Abai Kalikov on 4/22/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class StartGameTimerViewController: UIViewController {
+    
+    private enum Constants {
+        static let counterArray: [String] = ["3", "2", "1", ""]
+        static let lastRowDisplayIndex = 1
+        static let pickerViewRowHeight: CGFloat = 200
+        static let reusingViewFontSize: CGFloat = 140
+        static let timerTimeInterval: TimeInterval = 1.0
+    }
+    
+    @IBOutlet weak var timerPickerView: UIPickerView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configurePickerView()
+    }
+    
+    private func configurePickerView() {
+        timerPickerView.dataSource = self
+        timerPickerView.delegate = self
+        timerPickerView.isUserInteractionEnabled = false
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        makeTimer { [weak self] in
+            guard let self = self else { return }
+            self.performAnimatedTransition()
+        }
+    }
+    
+    private func performAnimatedTransition() {
+//                guard let window = UIApplication.shared.keyWindow else { return }
+//                window.rootViewController = UINavigationController(rootViewController: LoadingViewController())
+//                UIView.transition(with: window,
+//                                  duration: 0.7,
+//                                  options: .transitionFlipFromBottom,
+//                                  animations: {},
+//                                  completion: nil)
+        
+        //make transition to Game page
+    }
+    
+    private func makeTimer(completion: @escaping() -> Void) {
```

Comment:
makeTimer is not appropriate I think, configureCountdownTimer maybe

Reply:
Changed.